evalml/pipelines/components/transformers/feature_selection/rf_regressor_feature_selector.py

```diff
@@ -29,11 +28,11 @@ class RFRegressorSelectFromModel(FeatureSelector):
     name = "RF Regressor Select From Model"
     hyperparameter_ranges = {
         "percent_features": Real(0.01, 1),
-        "threshold": ["mean", -np.inf],
+        "threshold": ["mean"],
```

Comment:
Also here, seems to be ideal to allow `median` as a possible value in the hyperparams.